docs/source/science/ESA_CCI/ESA_CCI_V4.ipynb

```diff
@@ -7,7 +7,7 @@
     "tags": []
```

Comment:
<p>The only thing I'd do here is put the link with the additional resources at the top, or hyperlink it to make it a little cleaner. Maybe something like: "Now, we are going to filter the values which are less than 10 using the <code>.where</code> function ([DataArray.where Documentation]).</p><p> </p><p>Sorry, I know that's a very minor thing to point out!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Thank you, @smk0033 for your in-depth review. I have updated the tutorial to address your comments.